eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -54,7 +54,7 @@ stages:
             SubscriptionConfiguration: ${{ parameters.SubscriptionConfiguration }}
             CoverageReportPath: ${{ parameters.CoverageReportPath }}
 
-  - ${{ if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'), not(endsWith(variables['Build.DefinitionName'], ' - tests'))) }}:
+  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), not(endsWith(variables['Build.DefinitionName'], ' - tests'))) }}:
```

Comment:
Is this because you want to start having the scheduled build do start a release?

Reply:
The `Integration` stage in `archetype-cpp-release.yml` is a release. 

The `archetype-cpp-release.yml` already requires the build reason to be Manual -- https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/pipelines/templates/stages/archetype-cpp-release.yml#L8

The check in `archetype-sdk-client.yml` was redundant until I added the Integration stage and then it went from being redundant to being a bug.